src/content/interop/c-interop.md

```diff
@@ -259,7 +273,7 @@ use the [`package:ffigen`][ffigen] binding generator.
 
 <a id="native-assets" aria-hidden="true"></a>
 
-## Dart packages containing native code {: #build-hooks }
+## Depend on and bundle native code {: #build-hooks }
```

Comment:
I'd say it's either "Build and bundle native code" or "Build hooks and code assets"

Reply:
I like that better, thanks!